tap_intercom/stream.py

```diff
@@ -65,8 +65,13 @@ def do_sync(self, tap_stream_id: str, state: Optional[datetime] = None):
                 if errors:
                     code = errors[0].get("code")
                     message = errors[0].get("message")
-                    if code == "unauthorized" and message == "Access Token Invalid":
+
+                    token_suspended = code == "token_suspended" and message == "Unauthorized token, suspended application"
+                    unauthrized = code == "unauthorized" and message == "Access Token Invalid"
```

Comment:
Any reason for matching against the message? Seems fragile.

Reply:
only for belt and braces. I can do it with just code. There was in another tap think googleads or something where we check for invalid_grant. And there Patrick wanted me to double check for the message as well.